x/epochs/keeper/epoch_test.go

```diff
@@ -6,6 +6,60 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 )
 
+func (suite *KeeperTestSuite) TestAddEpochInfo() {
+	defaultIdentifier := "default_add_epoch_info_id"
+	defaultDuration := time.Hour
+	startBlockHeight := int64(100)
+	startBlockTime := time.Unix(1656907200, 0).UTC()
+	tests := map[string]struct {
+		addedEpochInfo types.EpochInfo
+		expErr         bool
+		expEpochInfo   types.EpochInfo
+	}{
+		"simple_add": {addedEpochInfo: types.EpochInfo{
+			Identifier:              defaultIdentifier,
+			StartTime:               time.Time{},
+			Duration:                defaultDuration,
+			CurrentEpoch:            0,
+			CurrentEpochStartHeight: 0,
+			CurrentEpochStartTime:   time.Time{},
+			EpochCountingStarted:    false,
+		}, expErr: false,
+			expEpochInfo: types.EpochInfo{
+				Identifier:              defaultIdentifier,
+				StartTime:               startBlockTime,
+				Duration:                defaultDuration,
+				CurrentEpoch:            0,
+				CurrentEpochStartHeight: startBlockHeight,
+				CurrentEpochStartTime:   time.Time{},
+				EpochCountingStarted:    false,
+			}},
+	}
+	for name, test := range tests {
+		suite.Run(name, func() {
+			suite.SetupTest()
+			suite.Ctx = suite.Ctx.WithBlockHeight(startBlockHeight).WithBlockTime(startBlockTime)
+			err := suite.App.EpochsKeeper.AddEpochInfo(suite.Ctx, test.addedEpochInfo)
+			if test.expErr != true {
+				suite.Require().NoError(err)
+				actualEpochInfo := suite.App.EpochsKeeper.GetEpochInfo(suite.Ctx, test.addedEpochInfo.Identifier)
+				suite.Require().Equal(test.expEpochInfo, actualEpochInfo)
+			} else {
+				suite.Require().Error(err)
+			}
```

Comment:
Are you planning to add more tests? I don't see one with an error expected?

Reply:
imo, I think the only notable errror for `AddEpochInfo` is the case for returning an error when there's a duplicate epoch identifier, which is tested in the test below